### DISCUSSION TITLE
Adding multiple labels to AutoscalingRunnerSet

### DISCUSSION THREAD

**Discussion by shakefu** [actions/actions-runner-controller#2820] [Questions]
Hi, I might've missed how to do this, but I noticed when creating a AutoscalingRunnerSet (in my case with the name "testing"), it would not pick up Jobs with any other `runs-on` labels, including `self-hosted`. 

Is there a way to add multiple labels to the scale sets so we can continue to use the self-hosted label (and others)?

**Comment by Link-** [comment]
Labels are not supported with the new autoscaling runner set mode. https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/about-actions-runner-controller#using-arc-runners-in-a-workflow
